tests/test_wave_03.py

```diff
@@ -55,12 +55,15 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
```

Comment:
This commented out code, particularly these items whose purpose is only to call your attention to add additional tests, would be totally fine to delete.